WooCommerce/Classes/Authentication/Jetpack Setup/Native Jetpack Setup/LoginJetpackSetupViewModel.swift

```diff
@@ -216,6 +227,7 @@ private extension LoginJetpackSetupViewModel {
     func checkJetpackConnection(retryCount: Int = 0) {
         guard retryCount <= Constants.maxRetryCount else {
             setupFailed = true
+            analytics.track(.loginJetpackSetupErrorCheckingJetpackConnection)
```

Comment:
Similar as above:
```suggestion
            analytics.track(.loginJetpackSetupErrorCheckingJetpackConnection, withError: error)
```

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/8187/commits/4edc245f1129f58d09830bafbc8f1bc156739244